packages/quantic/force-app/main/default/lwc/quanticTabBar/quanticTabBar.js

```diff
@@ -353,9 +357,11 @@ export default class QuanticTabBar extends LightningElement {
    * @returns {Array<Element>}
    */
   getTabsFromSlot() {
-    const isTab = (tagName) => /-quantic-tab$/i.test(tagName);
+    const isTab = (element) =>
```

Comment:
Slight nitpicking, but this code creates the `isTab` function every time the `getTabsFromSlot` is called, might be worth extracting it outside the function. Probably in a function of this class?